static/usage/v7/input/mask/javascript/index_html.md

```diff
@@ -0,0 +1,56 @@
+```html
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="https://cdn.skypack.dev/@ionic/core@7/css/core.css" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.skypack.dev/@ionic/core@7/css/ionic.bundle.css" />
+  </head>
```

Comment:
We do we add this again? Other JS code snippets just include whatever is in `ion-app`

Reply:
This JS playground uses multi-files and replaces the `index.html` file, so the CDN links needed to be respecified. This playground uses multi-files to import maskito and expose it on `window`, to be used in the template (same thing we have to do for the `modalController`, etc.)

Reply:
Oh right, thanks!